xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -1307,9 +1307,12 @@ static void DisplayReconfigured(CGDirectDisplayID display,
 {
   if (m_glView)
   {
-    dispatch_sync(dispatch_get_main_queue(), ^{
-      [m_glView NotifyContext];
-    });
+    if ([m_glView glContextOwned])
```

Comment:
this can be combined on the the first condition to avoid the nested if, will force push